Quotient/room.cpp

```diff
@@ -52,6 +52,7 @@
 #include "events/roomcanonicalaliasevent.h"
 #include "events/roomcreateevent.h"
 #include "events/roommemberevent.h"
+#include "events/roommessageevent.h"
```

Comment:
```suggestion
```
It's already included from `room.h` (but yeah, I slowly start thinking that include-what-you-use might ultimately be a good practice... but I'm not there yet).